src/admin/containers/email.js

```diff
@@ -76,9 +79,13 @@ const email = toggle_show => ([ address, { content, shown }]) =>
     >
       { shown ? 'Hide' : 'Show' } email
     </button>
-    { shown && <div>{ content.map(line => <p key={line}>{line}</p>) }</div> }
+    { shown && <div>{ content.map(displayEmail) }</div> }
   </li>
 
+const displayEmail = (line, i) => i === 0
```

Comment:
I don't love just using the head of the array to represent the subject. It would make more sense to me as a property on an object, or something like that. Not the end of the world though, this is still mergeable